capstone/src/main/java/com/google/sps/CloudNLPAssociation.java

```diff
@@ -67,8 +67,6 @@ public CloudNLPAssociation(LanguageServiceClient language) {
 
   @Override
   public void close() {
-    if (language != null) {
```

Comment:
Do you need to check to see if the client is null? If the class is initialized then the client is never null and this method can't be called unless the class is initialized since it lacks a static context.